shared/src/main/scala/scala/xml/pull/XMLEventReader.scala

```diff
@@ -24,6 +24,7 @@ import scala.xml.parsing.{ ExternalSources, MarkupHandler, MarkupParser }
  *  @author Burak Emir
  *  @author Paul Phillips
  */
+@deprecated("class XMLEventReader in package pull is deprecated(since 1.1.1):Consider javax.xml.stream.XMLEventReader instead.", "1.1.1")
```

Comment:
> Or am I still missing something here?

The deprecated tag message should just be `"Consider javax.xml.stream.XMLEventReader instead."`.  Currently, the message is duplicated by the compiler:

```
class XMLEventReader in package pull is deprecated (since 1.1.1): class XMLEventReader 
in package pull is deprecated(since 1.1.1):Consider javax.xml.stream.XMLEventReader instead.
```